crates/serde/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "prism-serde"
```

Comment:
I've looked into the preview for bincode 2.0.0 (its on RC.3 atm) and it looks like they will provide their own derives instead of using serde. Although they will provide backwards compatibility as a crate feature ("serde"), it is recommended for performance reasons, to move to their own derives. 

Now my point is: If we plan to make this crate contain all serialization code, we should probably replace serde with something more generic like:
transform, transcode, translate, convert, ...
Otherwise it might be confusing.

Reply:
I see serde as a clear, generic term for (de)serialization (instead of as a reference to the serde package), and I find the alternatives less precise and potentially more confusing